packages/context/src/context.ts

```diff
@@ -135,14 +140,15 @@ export class Context extends EventEmitter {
    * as the prefix
    * @param args - Arguments for the debug
    */
-  private _debug(...args: unknown[]) {
+  protected debugWithName(...args: unknown[]) {
```

Comment:
It makes me wonder, is it necessary to mention the fact that this method is prepending the context name in debug messages? Can we simply call it `debug`?

Reply:
Renamed to `debug`.